libs/packages/sam-formly/src/lib/formly-dialog/formly-dialog.component.ts

```diff
@@ -37,12 +41,12 @@ export class SdsFormlyDialogComponent implements OnInit {
 
   onSubmit() {
     if (this.form.valid) {
-      this.dialogRef.close(this.model);
+      this.submitFn.emit(this.model);
     }
```

Comment:
Shouldn't we always call the event and pass the form for them to check or allow consumer to do what they want? I think we should remove the conditional.

Reply:
Removed the condition.